src/index/updater.rs

```diff
@@ -128,15 +133,7 @@ impl<'index> Updater<'_> {
           // write transaction
           break;
         }
-        wtx
-          .open_table(WRITE_TRANSACTION_STARTING_BLOCK_COUNT_TO_TIMESTAMP)?
-          .insert(
-            &self.height,
-            &SystemTime::now()
-              .duration_since(SystemTime::UNIX_EPOCH)
-              .map(|duration| duration.as_millis())
-              .unwrap_or(0),
-          )?;
+        write_height_to_ts(&mut wtx, &self.height);
```

Comment:
Can you revert this change? What was the lint error here?

Reply:
![LugonShot 2024-02-17 at 13 19 58@2x](https://github.com/ordinals/ord/assets/48004924/3fff2346-52c9-4f0f-886b-f4554e47b1b8)